apps/bluetooth/js/modules/transfer_manager.js

```diff
@@ -175,7 +175,8 @@ define(function(require) {
     _onBluetoothEnabledChanged:
     function bttm__onBluetoothEnabledChanged(enabled) {
       Debug('_onBluetoothEnabledChanged(): enabled = ' + enabled);
-      if (enabled) {
+      // make sure the value is defined
```

Comment:
Simply `if (enabled === undefined) { return; }` here?

Do we want to run the `} else {` block if the value is `undefined`?